apitest/docs/api-beta-test-guide.md

```diff
@@ -0,0 +1,492 @@
+method help will be returned from the server.  Also note an api password is required to get help from the server.
+
+### Working With Encrypted Wallet
+
+There is no need to secure your regtest Bisq wallet with an encryption password when running these examples,
+but you should encrypt your mainnet wallet as you probably already do when using the Bisq UI to transact in
+real BTC.  This section explains how to encrypt your Bisq wallet with the CLI, and unlock it before performing wallet
+related operations such as creating and taking offers, checking balances, and sending BSQ and BTC to external wallets.
+
+Encrypt your wallet with a password:
+```
+$ ./bisq-cli --password=xyz setwalletpassword --wallet-password=<wallet-password>
+```
+
+Set a new password on your already encrypted wallet:
+```
+$ ./bisq-cli --password=xyz setwalletpassword --wallet-password=<wallet-password>
+    --new-wallet-password=<new-wallet-password>
+```
+
+Unlock your password encrypted wallet for N seconds before performing sensitive wallet operations:
+```
+$ ./bisq-cli --password=xyz unlockwallet --wallet-password=<wallet-password> --timeout=<seconds>
+```
+You can override a `timeout` before it expires by calling `unlockwallet` again.
+
+Lock your wallet before the `unlockwallet` timeout expires:
+```
+$ ./bisq-cli --password=xyz lockwallet
+```
+
+### Checking Balances
+
+Show full BSQ and BTC wallet balance information:
+```
+$ ./bisq-cli --password=xyz --port=9998  getbalance
+```
+
+Show full BSQ wallet balance information:
+```
+$ ./bisq-cli --password=xyz --port=9999 getbalance --currency-code=bsq
+```
+_Note:  The example above is asking for Bob’s balance (using port `9999`), not Alice’s balance._
+
+Show Bob’s full BTC wallet balance information:
+```
+$ ./bisq-cli --password=xyz --port=9999 getbalance --currency-code=btc
+```
+
+### Funding a Bisq Wallet
+
+#### Receiving BTC
+
+To receive BTC from an external wallet, find an unused BTC address (with a zero balance) to receive the BTC.
+```
+$ ./bisq-cli --password=xyz --port=9998 getfundingaddresses
+```
+You can check block explorers for the status of a transaction, and check your Bisq BTC wallet address directly:
+```
+$ ./bisq-cli --password=xyz --port=9998 getaddressbalance --address=<btc-address>
+```
+
+#### Receiving BSQ
+To receive BSQ from an external wallet, find an unused BSQ address:
+```
+$ ./bisq-cli --password=xyz --port=9998 getunusedbsqaddress
+```
+
+Give the public address to the sender.  After the BSQ is sent, you can check block explorers for the status of
+the transaction.  There is no support (yet) to check the balance of an individual BSQ address in your wallet,
+but you can check your BSQ wallet’s balance to see when the new funds arrive:
+```
+$ ./bisq-cli --password=xyz --port=9999 getbalance --currency-code=bsq
+```
+
+### Sending BSQ and BTC to External Wallets
+
+Below are commands for sending BSQ and BTC to external wallets.
```

Comment:
Sending BSQ to a non-Bisq wallet is not supported and is likely to cause problems and/or losses...maybe a warning like the following could be worth adding here?

"Please note that sending BSQ to non-Bisq wallets is not supported and highly discouraged."